### ISSUE TITLE
make down command not working

### ISSUE THREAD

**Issue by vkobozev** [sherifabdlnaby/elastdocker#47]
**Describe the bug**
'make down' command not working

**To Reproduce**
```
elastdocker$ make down
ERROR: The Compose file './docker-compose.tools.yml' is invalid because:
services.rubban.environment.RUBBAN_AUTOINDEXPATTERN_ENABLED contains true, which is an invalid type, it should be a string, number, or a null
Makefile:61: recipe for target 'down' failed
make: *** [down] Error 1
```

**Solution:**
edit `./docker-compose.tools.yml`:
currently - `RUBBAN_AUTOINDEXPATTERN_ENABLED: true`
should be - `RUBBAN_AUTOINDEXPATTERN_ENABLED: 'true'`

BR,
Vasily.

**Comment by sherifabdlnaby** [comment]
@vkobozev Weird, it's not reproducible in my machine. Maybe it has to do with the docker-compose version. Mind if you share your docker and docker-compose version?

I'll push a fix anyway though.

**Comment by vkobozev** [comment]
Here you are.
Maybe it is regarding the versions.
Anyway, I would appreciate if you consider to add quotes for compatibility.

```$docker-compose version
docker-compose version 1.29.1, build c34c88b2
docker-py version: 5.0.0
CPython version: 3.7.10
OpenSSL version: OpenSSL 1.1.0l  10 Sep 2019

$ docker version
Client: Docker Engine - Community
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        370c289
 Built:             Fri Apr  9 22:46:01 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.6
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8728dd2
  Built:            Fri Apr  9 22:44:13 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.4
  GitCommit:        05f951a3781f4f2c1911b05e61c160e9c30eaa8e
 runc:
  Version:          1.0.0-rc93
  GitCommit:        12644e614e25b05da6fd08a38ffa0cfe1903fdec
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0```

**Comment by sherifabdlnaby** [comment]
@vkobozev Fixed in PR #48
